src/Nethermind/Nethermind.Blockchain.Test/BlockTreeTests.cs

```diff
@@ -45,7 +46,7 @@ private BlockTree BuildBlockTree()
             _blocksDb = new MemDb();
             _headersDb = new MemDb();
             _blocksInfosDb = new MemDb();
-            return new BlockTree(_blocksDb, _headersDb, _blocksInfosDb, new ChainLevelInfoRepository(_blocksInfosDb),  MainNetSpecProvider.Instance, NullTxPool.Instance, LimboLogs.Instance);
+            return new BlockTree(_blocksDb, _headersDb, _blocksInfosDb, new ChainLevelInfoRepository(_blocksInfosDb),  MainNetSpecProvider.Instance, NullTxPool.Instance, NullBloomStorage.Instance, LimboLogs.Instance);
```

Comment:
looks like some Bloom management was added to BlockTree and there are no new tests